n_cjson_helpers.c

```diff
@@ -431,15 +431,18 @@ const char *JGetItemName(const J * item)
 void JItoA(long int n, char *s)
 {
     char c;
-    long int i, j, sign;
-    if ((sign = n) < 0) {
-        n = -n;
+    // Conversion to unsigned is required to handle the case where n is
+    // LONG_MIN.
+    unsigned long int unsignedN = n;
+    long int i, j;
+    if (n < 0) {
+        unsignedN = -unsignedN;
```

Comment:
The data type of `unisignedN` is `unsigned long int`, so what does making it negative do?

Reply:
See above.